e2e/gui/src/test/java/com/epam/pipeline/autotests/ao/SettingsPageAO.java

```diff
@@ -1026,6 +1027,22 @@ public PreferencesAO setSystemSshDefaultRootUserEnabled() {
             return this;
         }
 
+        public String[] getAmisFromClusterNetworksConfigPreference() {
+            searchPreference("cluster.networks.config");
+            String[] result = new String[2];
+            String[] strings = context().$(byClassName("CodeMirror-code"))
+                            .findAll(byClassName("CodeMirror-line")).texts().stream()
+                            .toArray(String[]::new);
```

Comment:
Can be replaced with collection.toArray()

Reply:
Done